twake/frontend/src/app/components/LockedFeaturesComponents/LockedGuestsPopup/LockedGuestsPopup.tsx

```diff
@@ -0,0 +1,42 @@
+import { Button } from 'antd';
+import ObjectModal from 'app/components/ObjectModal/ObjectModal';
+import React from 'react';
+import ModalManager from 'app/components/Modal/ModalManager';
+import Languages from 'services/languages/languages.js';
+import RouterService from 'app/services/RouterService';
+import Emojione from 'app/components/Emojione/Emojione';
+
+const LockedGuestsPopup = (): JSX.Element => {
+  const { companyId } = RouterService.getStateFromRoute();
+  const pricingPlanUrl = `https://console.twake.app/companies/${companyId}/subscription`;
```

Comment:
This should come from config so from props

Reply:
Done 👍🏻